Gems/EMotionFX/Code/Source/Editor/SkeletonModelJointWidget.cpp

```diff
@@ -165,6 +165,19 @@ namespace EMotionFX
         return actor;
     }
 
+    ActorInstance* SkeletonModelJointWidget::GetActorInstance() const
```

Comment:
update: This is `const` broken - we should either return a `const ActorInstance*` or remove the `const` from this member function (as we have a `const` function handing out a non-`const` resource)

Reply:
Removed the const.